### PR TITLE
Moviepage

### PR DESCRIPTION
**Pull request by LillanEriksson**
Link to my deployed project:
https://condescending-fermat-253ba9.netlify.app/

My reflections on how this week's project turned out:
I'm really satisfied with the outcome of this project, it was fun and challenging. and I felt I actually starting to grasp all that we learnt until now!

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import './header.css';
+
+export const Header = () => {
+	return (
+		<header>
+			<h1>Welcome to the moooooviepage</h1>
+		</header>
+	);
+};
```
Comment (Thessan) [inline code/src/components/Header.js:11]: I can't see this header being displayed on the page?

File: code/src/components/movie.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	position: relative;
+	width: 100%;
+}
+
+.moviewrapper__description {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0px;
+	left: 0px;
```
Comment (Thessan) [inline code/src/components/movie.css:16]: No need for px after 0 ☺️

File: code/src/pages/MovieList.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+	return (
+		<section className="movielist__container">
+			{loading && <Spinner />}
+			{movies.map((movie) => (
+				<Movie key={movie.id} {...movie} />
+			))}
+		</section>
+	);
+};
```
Comment (Thessan) [inline code/src/pages/MovieList.js:30]: Nice code, easy to read 👏

File: code/src/components/Button.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30">
+				<path
+					d="M27 14.5C27 7.596441 21.4035594 2 14.5 2S2 7.596441 2 14.5 7.5964406 27 14.5 27 27 21.403559 27 14.5zm-19.3388348-.353553l7.4852814-7.485282c.1952622-.195262.5118446-.195262.7071068 0l2.1213203 2.121321c.1952622.195262.1952622.511844 0 .707106L12.9644661 14.5l5.0104076 5.010408c.1952622.195262.1952622.511844 0 .707106l-2.1213203 2.121321c-.1952622.195262-.5118446.195262-.7071068 0l-7.4852814-7.485282c-.19799-.19799-.197989-.509117 0-.707106z"
+					fill="#fff"
+					fillRule="evenodd"></path>
+			</svg>
+			<animated.h2 style={animatedTextStyle}>Movies</animated.h2>
+		</button>
+	);
+};
```
Comment (Thessan) [inline code/src/components/Button.js:38]: Nice job trying out the animation 👍 However, the button does not direct you back to the movie list page while being clicked (tried it on both Safari and Chrome).

File: code/src/components/button.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
+button {
+	margin: 20px;
+	background: transparent;
+	color: white;
+	font-size: 20px;
+	font-weight: bold;
+	border-style: none;
+	display: flex;
+	align-items: center;
+}
```
Comment (Thessan) [inline code/src/components/button.css:10]: Think about indentation ☺️

File: code/src/pages/MovieInfo.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+	useEffect(() => {
+		fetch(
+			`https://api.themoviedb.org/3/movie/${id}?api_key=1cd52a160617841675a7339879fe0ce3&language=en-US`
+		)
+			.then((res) => res.json())
+			.then((data) => setMovie(data));
+	}, [id]);
+
+	//trying out React Spring for the button-text animation. Could have worked better!
```
Comment (Thessan) [inline code/src/pages/MovieInfo.js:20]: Maybe place this comment on the Button page instead? ☺️

File: code/src/pages/PageNotFound.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+		<div className="page-not-found">
+			<section className="page-not-found__error">
+				<h1>Something went wrong...</h1>
+				<h2>Go back to the movies page and pick another movie.</h2>
+
+				<Button />
+			</section>
+		</div>
+	);
+};
```
Comment (Thessan) [inline code/src/pages/PageNotFound.js:18]: I've been trying to reach this custom page, but seems like I can only get to the original one?

**Review by Thessan — CHANGES_REQUESTED** [review]
Good job Lillemor! 👏 The page looks nice and you did a good job on the animation of the button. Please take a look however on the function of the button, so it re-directs you back to the movie list ☺️

File: code/src/components/Header.js (hunk 1 above)
Comment (LillanEriksson) [reply code/src/components/Header.js]: Removed it now, because it added nothing to the page really...

File: code/src/components/Button.js (hunk 4 above)
Comment (LillanEriksson) [reply code/src/components/Button.js]: Now it does! :)

File: code/src/components/button.css (hunk 5 above)
Comment (LillanEriksson) [reply code/src/components/button.css:10]: Fooork... I can't get the indentation to show up nice in github, it looks great in VSCode...

File: code/src/pages/PageNotFound.js (hunk 7 above)
Comment (LillanEriksson) [reply code/src/pages/PageNotFound.js]: Yayy got it working now!

File: code/src/components/Movie.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+import './movie.css';
+
+export const Movie = ({ id, title, release_date, poster_path }) => {
+	const [showTitle, setShowTitle] = useState(false);
+	return (
+		<Link
+			to={`/movies/${id}`}
+			onMouseOver={() => setShowTitle(true)}
+			onMouseLeave={() => setShowTitle(false)}>
```
Comment (EmOlsz) [inline code/src/components/Movie.js:12]: Great job!

**Pull request closed without merging**